src/components/App/AppLogo.jsx

```diff
@@ -17,7 +17,7 @@ const StyledLink = styled(Link)`
 `;
 
 const AppLogo = ({ size, strings, onClick }) => (
-  <StyledLink to="/" onClick={onClick}>
+  <StyledLink aria-label="Go to the Open Dota homepage" to="/" onClick={onClick}>
```

Comment:
This is needed to provide clarity to screen reader users. Currently because css is used to transform the text to uppercase, screen readers spell out the link as `Less than, O-P-E-N-D-O-T-A backslash greater than`. With this aria label screen readers will now provide more accurate context to users who utilize them. 

<details><summary>VoiceOver</summary>
<img width="597" alt="Screen Shot 2021-10-02 at 10 22 48 AM" src="https://user-images.githubusercontent.com/15097156/135722754-811dc549-30ea-4e37-9226-dc3fc14f7cba.png">

</details>